docs/2.drivers/netlify.md

```
@@ -4,14 +4,14 @@ icon: teenyicons:netlify-solid
 
 # Netlify Blobs
 
-> Store data in Netlify Blobs.
+> 在 Netlify Blobs 中存储数据。
 
-Store data in a [Netlify Blobs](https://docs.netlify.com/blobs/overview/) store. This is supported in both [edge](#using-in-netlify-edge) and Node.js function runtimes, as well as during builds.
+在 [Netlify Blobs](https://docs.netlify.com/blobs/overview/) 存储中存储数据。这在 [边缘](#using-in-netlify-edge) 和 Node.js 函数运行时中都得到支持，以及在构建期间。
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)